packages/hash/frontend/src/assets/svg/lightbulb-on.svg

```diff
@@ -1,12 +1,5 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none"
-  xmlns="http://www.w3.org/2000/svg">
-  <g opacity="0.5">
-    <path fill-rule="evenodd" clip-rule="evenodd" d="M15.1875 10.3125C15.1875 7.62007 13.0049 5.4375 10.3125 5.4375C7.62007 5.4375 5.4375 7.62007 5.4375 10.3125C5.4375 13.0049 7.62007 15.1875 10.3125 15.1875C11.4523 15.1875 12.5008 14.7963 13.331 14.1408C13.419 14.0714 13.5455 14.0758 13.6247 14.155V14.155C13.6649 14.1952 13.6875 14.2498 13.6875 14.3066V14.5152C13.6875 14.7055 13.7631 14.8881 13.8977 15.0227L16.875 18C17.1857 18.3107 17.6893 18.3107 18 18V18C18.3107 17.6893 18.3107 17.1857 18 16.875L15.0227 13.8977C14.8881 13.7631 14.7055 13.6875 14.5152 13.6875H14.3066C14.2498 13.6875 14.1952 13.6649 14.155 13.6247V13.6247C14.0758 13.5455 14.0714 13.419 14.1408 13.331C14.7963 12.5008 15.1875 11.4523 15.1875 10.3125ZM6.9375 10.3125C6.9375 8.44852 8.44852 6.9375 10.3125 6.9375C12.1765 6.9375 13.6875 8.44852 13.6875 10.3125C13.6875 12.1765 12.1765 13.6875 10.3125 13.6875C8.44852 13.6875 6.9375 12.1765 6.9375 10.3125Z" fill="black"/>
-    <mask id="mask0_260332:1375" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="5" y="5" width="14" height="14">
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M15.1875 10.3125C15.1875 7.62007 13.0049 5.4375 10.3125 5.4375C7.62007 5.4375 5.4375 7.62007 5.4375 10.3125C5.4375 13.0049 7.62007 15.1875 10.3125 15.1875C11.4523 15.1875 12.5008 14.7963 13.331 14.1408C13.419 14.0714 13.5455 14.0758 13.6247 14.155V14.155C13.6649 14.1952 13.6875 14.2498 13.6875 14.3066V14.5152C13.6875 14.7055 13.7631 14.8881 13.8977 15.0227L16.875 18C17.1857 18.3107 17.6893 18.3107 18 18V18C18.3107 17.6893 18.3107 17.1857 18 16.875L15.0227 13.8977C14.8881 13.7631 14.7055 13.6875 14.5152 13.6875H14.3066C14.2498 13.6875 14.1952 13.6649 14.155 13.6247V13.6247C14.0758 13.5455 14.0714 13.419 14.1408 13.331C14.7963 12.5008 15.1875 11.4523 15.1875 10.3125ZM6.9375 10.3125C6.9375 8.44852 8.44852 6.9375 10.3125 6.9375C12.1765 6.9375 13.6875 8.44852 13.6875 10.3125C13.6875 12.1765 12.1765 13.6875 10.3125 13.6875C8.44852 13.6875 6.9375 12.1765 6.9375 10.3125Z" fill="white"/>
-    </mask>
-    <g mask="url(#mask0_260332:1375)">
-      <rect width="24" height="24" fill="#1B1D24"/>
-    </g>
-  </g>
-</svg>
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    d="M15.9736 14.7188L11.7861 10.5312C12.6924 9.4375 13.1924 8.03125 13.1924 6.5C13.1924 2.9375 10.2549 0 6.69238 0C3.09863 0 0.223633 2.9375 0.223633 6.5C0.223633 10.0938 3.12988 13 6.69238 13C8.19238 13 9.59863 12.5 10.7236 11.5938L14.9111 15.7812C15.0674 15.9375 15.2549 16 15.4736 16C15.6611 16 15.8486 15.9375 15.9736 15.7812C16.2861 15.5 16.2861 15.0312 15.9736 14.7188ZM1.72363 6.5C1.72363 3.75 3.94238 1.5 6.72363 1.5C9.47363 1.5 11.7236 3.75 11.7236 6.5C11.7236 9.28125 9.47363 11.5 6.72363 11.5C3.94238 11.5 1.72363 9.28125 1.72363 6.5Z"
+    fill="#91A5BA" />
+</svg>
```

Comment:
I have the same suggestion as mentioned in https://github.com/hashintel/hash/pull/358/files#r817643896